Specs/Core/OrientedBoundingBoxSpec.js

```diff
@@ -338,6 +338,9 @@ defineSuite([
             Matrix3.multiplyByVector(axes, tang, tang);
             Matrix3.multiplyByVector(axes, binorm, binorm);
             Cartesian3.cross(tang, binorm, n);
+            if (Cartesian3.magnitude(n) === 0) {
+                return;
```

Comment:
Can you use `return undefined;` here for consistency with the rest of the function.

The rule we tend to use (but doubt we've ever codified) is that we use `return undefined;` in cases where the function returns actual values sometimes as well.